core/crypto/node_ks.go

```diff
@@ -149,6 +149,10 @@ func (ks *keyStore) storePrivateKeyInClear(alias string, privateKey interface{})
 	return nil
 }
 
+func (ks *keyStore) deletePrivateKeyInClear(alias string) error {
+	return os.Remove(ks.node.conf.getPathForAlias(alias))
```

Comment:
Is the `alias` in any way user input? Is there any need to be concerned about directory traversal attacks?